platform_it/pom.xml

```diff
@@ -1180,6 +1184,14 @@
                 <extraDependency2>splice_ee</extraDependency2>
                 <extraDependency3>splice_ee</extraDependency3>
             </properties>
+            <dependencies>
```

Comment:
This is a wrong place to add dependencies for _dbaas3.0_. This sections is instrumental for the _ee_ profile below.

Reply:
yes, wrong place, spark-avro for dbaas3 profile is moved to hbase_sql, where it's configured already for other profiles